player/command.c

```diff
@@ -1042,19 +1042,38 @@ static int mp_property_edition(void *ctx, struct m_property *prop,
 {
     MPContext *mpctx = ctx;
     struct demuxer *demuxer = mpctx->demuxer;
+    int ed;
+    char *name = NULL;
 
-    if (action == M_PROPERTY_GET_CONSTRICTED_TYPE && demuxer) {
-        if (demuxer->num_editions <= 1)
-            return M_PROPERTY_UNAVAILABLE;
+    if (!demuxer)
+        return mp_property_generic_option(mpctx, prop, action, arg);
+
+    ed = demuxer->edition;
+
+    if (demuxer->num_editions <= 1)
+        return M_PROPERTY_UNAVAILABLE;
+
+    switch (action) {
+    case M_PROPERTY_GET_CONSTRICTED_TYPE: {
         *(struct m_option *)arg = (struct m_option){
             .type = CONF_TYPE_INT,
             .min = 0,
             .max = demuxer->num_editions - 1,
         };
         return M_PROPERTY_OK;
     }
-
-    return mp_property_generic_option(mpctx, prop, action, arg);
+    case M_PROPERTY_PRINT: {
+        if (ed < 0)
+            return M_PROPERTY_UNAVAILABLE;
+        name = mp_tags_get_str(demuxer->editions[ed].metadata, "title");
```

Comment:
If you return this without copying the string, you'll get a dangling pointer in the tags struct.